### PR TITLE
Minor changes and descriptions

### PR DESCRIPTION
**Pull request by zdravko-g**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+			pwsh ('''
+					nuget restore "${env:workspace}/ADotNet.sln"
+				''')
+				bat "\"${tool 'MSBuild 17'}/msbuild.exe\" \"${env:workspace}/ADotNet.sln\" /clp:ErrorsOnly -t:build -property:Configuration=Release -property:Version=$buildVersion -property:PackageVersion=$buildVersion -property:AssemblyVersion=$buildVersion"
+			}
+		}
+		
+		stage('Push nuget packages') {
+			steps {
+				withCredentials([string(credentialsId: 'nuget-token', variable: 'TOKEN')]) {
```
Comment (zdravko-g) [inline Jenkinsfile:23]: @nikolayrk do you know if this is the correct way to get the token?

Comment (nikolayrk) [reply Jenkinsfile:23]: [It is - yeah](https://www.jenkins.io/doc/pipeline/steps/credentials-binding/)

Comment (nikolayrk) [reply Jenkinsfile:23]: Make sure the credential is of type **[Secret text](https://www.jenkins.io/doc/book/using/using-credentials/)**, which I'm assuming is the appropriate type mentioned here:

image

**Review by nikolayrk — APPROVED** [review]
LGTM

**Pull request merged**